site/docs/foundations/spacing.mdx

```diff
@@ -25,34 +27,36 @@ Every space token within the system is a multiple of `--salt-spacing-100`. The n
 | `--salt-spacing-300` | 3.00x   | 12                | 24                  | 36               | 48                 |
 | `--salt-spacing-350` | 3.50x   | 14                | 28                  | 42               | 56                 |
 | `--salt-spacing-400` | 4.00x   | 16                | 32                  | 48               | 64                 |
+| `--salt-spacing-450` | 4.50x   | 18                | 36                  | 54               | 72                 |
```

Comment:
These aren't currently in the system.

We should either remove them temporarily or hold off merging this PR until they are added.

Reply:
Hopefully :)